kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/dhcp/server/DnsmasqTool.java

```diff
@@ -88,6 +88,7 @@ public CommandStatus startInterface(String interfaceName) throws KuraProcessExec
         CommandStatus restartStatus = this.executorService.execute(RESTART_COMMAND);
 
         if (!restartStatus.getExitStatus().isSuccessful()) {
+            logger.error("dnsmasq Systemd unit startup failed. Is dnsmasq package installed on the system?");
```

Comment:
The error here might also be due to another config file that uses an inconsitent setup (like defining the `bind-interfaces` property). I would remove this `Is dnsmasq package installed on the system?` and print the error stream instead